plugins/media-keys/msd-media-keys-manager.c

```diff
@@ -487,9 +496,9 @@ do_url_action (MsdMediaKeysManager *manager,
         if (app_info != NULL) {
            if (!g_app_info_launch (app_info, NULL, NULL, &error)) {
                 g_warning ("Could not launch '%s': %s",
-                    g_app_info_get_commandline (app_info),
-                    error->message);
-		g_object_unref (app_info);
+                           g_app_info_get_commandline (app_info),
+                           error->message);
+                g_object_unref (app_info);
```

Comment:
If you do not make any changes to the code, please do not format it, as you will increase the number of lines to review.